plugins/base/src/test/kotlin/content/seealso/ContentForSeeAlsoTest.kt

```diff
@@ -517,76 +517,73 @@ class ContentForSeeAlsoTest : BaseAbstractTest() {
     }
 
     @Test
-    fun `undocumented see also from java`(){
```

Comment:
Moved two tests to `JavadocParserTest` as they are not about content, but verify documentables model.